src/ssltunnel.js

```diff
@@ -77,10 +77,17 @@ const TunnelService = {
   start: function(response, urlredirect) {
     Settings.get('tunneltoken').then((result) => {
       if (typeof result === 'object') {
+        if (!result.base) {
+          // handle legacy tunnels
+          result.base = 'mozilla-iot.org';
```

Comment:
Should this be set as a constant somewhere?

Reply:
Potentially. This code will probably be removed pretty quickly, but I may need to add a constant later to handle the rest of the transition logic.

Reply:
Yeah I'm looking forward to ripping all this transition code out again as soon as possible!